spec/helpers/spec-helper.coffee

```diff
@@ -1,6 +1,18 @@
-window.App ?= {}
-window.App.Models ?= {}
-window.App.Collections ?= {}
+$ = require 'jquery'
+_ = require 'underscore'
+Backbone = require 'backbone'
+
+jqueryMatchers = require 'jasmine-jquery-matchers'
+BackboneFactory = require 'backbone-factory'
+
+StorageManager = require '../../src/storage-manager'
+
+TimeEntries = require './models/time-entries'
+Posts = require './models/posts'
+Tasks = require './models/tasks'
+Projects = require './models/projects'
+Users = require './models/users'
+
 
 window.resultsArray = (key, models) ->
```

Comment:
Couldn't we make this something that we export rather than placed on the window?